ush/rocoto/setup_expt.py

```diff
@@ -135,6 +136,7 @@ def edit_baseconfig():
 link initial condition files from $ICSDIR to $COMROT'''
 
     parser = ArgumentParser(description=description, formatter_class=ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--app', help='UFS application', type=str, choices=['ATM', 'ATMW', 'S2S', 'S2SW'], required=False, default='ATM')
```

Comment:
Not request for a change, just a thought...I imagine this list of possible APPs will show itself in multiple places, is there a way to consolidate all of that? For example, the value of `choices` here could be sourced from some sort of common file/place earlier in the script. Have been contemplating something similar for a while to handle the list of supported machines/target that's listed/used through the system and other similar lists that will grow/shrink over time as we support more/less. Food for thought, thanks!